src/pykeen/models/__init__.py

```diff
@@ -85,9 +83,12 @@
     'make_model_cls',
 ]
 
-_MODELS: Set[Type[Model]] = {
-    subcls
-    for subcls in get_subclasses(Model)  # type: ignore
-    if not subcls._is_base_model
-}
-model_resolver = Resolver(classes=_MODELS, base=Model)  # type: ignore
+model_resolver = Resolver.from_subclasses(
```

Comment:
:heart_eyes:

Reply:
This was an excellent addition to `class-resolver` from you ❤️